source/views/dictionary/list.tsx

```diff
@@ -104,32 +114,32 @@ export function DictionaryView(props: Props) {
 	)
 
 	return (
-		<View style={styles.wrapper}>
-			<SearchBar onChange={setQuery} value={query} />
-
-			<SectionList
-				ItemSeparatorComponent={ListSeparator}
-				ListEmptyComponent={
-					<NoticeView text={`No results found for "${query}"`} />
-				}
-				contentContainerStyle={styles.contentContainer}
-				keyExtractor={(item: WordType, index) => item.word + index}
-				keyboardDismissMode="on-drag"
-				keyboardShouldPersistTaps="never"
-				refreshControl={
-					<RefreshControl onRefresh={reload} refreshing={isPending} />
-				}
-				renderItem={renderRow}
-				renderSectionHeader={({section: {title}}) => (
-					<ListSectionHeader title={title} />
-				)}
-				sections={grouped}
-				style={styles.wrapper}
-			/>
-		</View>
+		<SectionList
+			ItemSeparatorComponent={ListSeparator}
+			ListEmptyComponent={
+				<NoticeView text={`No results found for "${query}"`} />
+			}
+			contentContainerStyle={styles.contentContainer}
+			contentInsetAdjustmentBehavior="automatic"
+			keyExtractor={(item: WordType, index) => item.word + index}
+			keyboardDismissMode="on-drag"
+			keyboardShouldPersistTaps="never"
+			refreshControl={
+				<RefreshControl onRefresh={reload} refreshing={isPending} />
+			}
+			renderItem={renderRow}
+			renderSectionHeader={({section: {title}}) => (
+				<ListSectionHeader title={title} />
+			)}
+			sections={grouped}
+			style={styles.wrapper}
+		/>
 	)
 }
-DictionaryView.navigationOptions = {
+
+export {DictionaryView as View}
+
+export const NavigationOptions: NativeStackNavigationOptions = {
 	title: 'Campus Dictionary',
-	headerBackTitle: 'Dictionary',
+	headerBackTitle: 'Back',
```

Comment:
Why?

Reply:
I can't find the exact documentation I read but the API is (re)written such that the navigation options reflect the current route and the current route only.

If we were to keep "Dictionary" as the back title, we would push to the dictionary list, and the back title would be "Dictionary" which doesn't reflect that back is going to the Home view.

Reply:
Okay, I was wondering if that was the case!